src/Elcodi/Admin/MediaBundle/Controller/ImageController.php

```diff
@@ -144,12 +145,20 @@ public function disableAction(
     public function deleteAction(
         Request $request,
         $entity,
-        $redirectUrl = null
+        $redirectPath = null
     ) {
+        $redirectPathParam = $request
```

Comment:
This can be removed if https://github.com/mmoreram/ControllerExtraBundle/pull/55 is approved.

The action will become:
```php
<?php
//...

/**
 * ...
 * @Get(
 *    path = "redirect-path",
 *    name = "redirectPath"
 * )
 */
public function deleteAction(
    Request $request,
    $entity,
    $redirectPath = null
) {
    $redirectPath = is_null($redirectPath)
        ? $this->generateUrl('admin_image_list')
        : $redirectPath;

    return parent::deleteAction(
        $request,
        $entity,
        $redirectPath
    );
}
```